package.json

```diff
@@ -4,16 +4,43 @@
   "description": "OrbitDB Benchmark Runner",
   "main": "./src/index.js",
   "bin": {
-    "benchmark-runner": "./src/cli.js"
+    "benchmarker": "./src/cli.js"
```

Comment:
Should we change the name of the repo to `orbit-db-benchmarker`? Thinking about consistency.

Reply:
this could be changed to anything or kept the same, i chose the different name originally so it didnt conflict. probably good to have the package and repo the same name though (they are different right now; repo: benchmark-runner, package: orbit-db-benchmark-runner). shorter name probably better. also keep in mind the benchmarker doesnt have anything written in that makes it specific to benchmarking orbitdb, it can run benchmarks for anything.

Reply:
Alright let's just leave it as is for now until it becomes a real problem